automation/jenkins_build.sh

```diff
@@ -218,11 +218,17 @@ if [ "$deploy" == "yes" ]; then
 		-e SLUG="$SLUG" \
 		-e BUILD_VERSION="$S3_VERSION_HOSTOS" \
 		-e DEVELOPMENT_IMAGE="$DEVELOPMENT_IMAGE" \
+		-e DEPLOYER_UID=$(id -u) \
+		-e DEPLOYER_GID=$(id -g) \
 		-v $S3_DEPLOY_DIR:/host/images resin/resin-img:master /bin/sh -x -e -c ' \
+			echo "Creating and setting deployer user $DEPLOYER_UID:$DEPLOYER_GID."
+			groupadd -g $DEPLOYER_GID deployer
+			useradd -m -u $DEPLOYER_UID -g $DEPLOYER_GID deployer
+			sudo su deployer
 			echo "${BUILD_VERSION}" > "/host/images/${SLUG}/latest"
 			/usr/src/app/node_modules/.bin/coffee /usr/src/app/scripts/prepare.coffee
-			apt-get -y update
-			apt-get install -y s3cmd
+			sudo apt-get -y update
```

Comment:
Correct. I'm wondering if there is any gain from it - as it's still two lines change.